staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go

```diff
@@ -128,7 +128,10 @@ func (r *rudimentaryErrorBackoff) OnError(error) {
 	r.lastErrorTimeLock.Lock()
 	defer r.lastErrorTimeLock.Unlock()
 	d := time.Since(r.lastErrorTime)
-	if d < r.minPeriod {
+	if d < 0 {
+		// reset when time hopping
+		time.Sleep(r.minPeriod)
```

Comment:
Sleeping is probably not necessary in this case--it should be rare, and the point of this is just to avoid hot loops.